stable/rabbitmq-ha/values.yaml

```diff
@@ -223,3 +223,24 @@ serviceAccount:
   ## The name of the ServiceAccount to use.
   ## If not set and create is true, a name is generated using the fullname template
   # name:
+
+secret:
```

Comment:
Could keep the file consistent and add descriptions for fields that require it?
Also, the README.md should be updated if you add new fields.